presto-main/src/main/java/com/facebook/presto/sql/planner/CompilerConfig.java

```diff
@@ -23,6 +24,8 @@
 public class CompilerConfig
 {
     private int expressionCacheSize = 10_000;
+    private int scanNodeLimit = 100_000;
```

Comment:
Did you check if existing queries don't exceed this threshold? They probably do because of our stage limit, though. Ping me on how we can check this

Reply:
Changed to 10_000 which is closer to realistic limit (although still a bit conservative), and I assume that we will run dark traffic tune the real number later)